pokemongo_bot/logger.py

```diff
@@ -6,14 +7,18 @@
 except:
     lcd = False
 
-
-def log(string, color='white'):
-    colorHex = {'green': '92m', 'yellow': '93m', 'red': '91m'}
-
+def log(string, color = 'white'):
+    colorHex = {
+        'green': '92m',
+        'yellow': '93m',
+        'red': '91m'
+    }
     if color not in colorHex:
+        print(time.strftime("%Y-%m-%d %H:%M:%S"))
         print(string)
```

Comment:
This will results in eg:
```
2016-07-23 20:59:33
string
```
Can you make like like 
```
[2016-07-23 20:59:33] string
```
Will be much cleaner then
Corrected code:
```
print(time.strftime("%Y-%m-%d %H:%M:%S") + ' '+ string)
```